processing/db_handler.py

```diff
@@ -260,6 +260,7 @@ def register_format_qaqc(self, upload_id: int, process_timestamp: str,
 
         conn = self.init_db_conn(db_config)
         process_id = self._register_qaqc_process(conn, field_names, values)
+        conn.close()
```

Comment:
Probably a good idea to do the whole `with` or `try/except/finally` for handling DB connection closures

Reply:
This will be removed in next PR.

Reply:
More detail: conn will be passed in as parameter.
Eventually a webservice will replace the direct write to the DB.